coordinator/src/services/validation/claim-validator.ts

```diff
@@ -1,15 +1,27 @@
 import {fhir, validationErrors as errors} from "@models"
 import {validatePermittedDispenseMessage} from "./scope-validator"
 
-export function verifyClaim(claim: fhir.Claim, scope: string): Array<fhir.OperationOutcomeIssue> {
+export function verifyClaim(
+  claim: fhir.Claim, scope: string, accessTokenOds: string
+): Array<fhir.OperationOutcomeIssue> {
+  const validationErrors = []
   if (claim.resourceType !== "Claim") {
-    return [errors.createResourceTypeIssue("Claim")]
+    validationErrors.push(errors.createResourceTypeIssue("Claim"))
```

Comment:
I think this was returning deliberately because if it's not a Claim, we can't really validate anything else

Reply:
Same in other places